xwiki-commons-core/xwiki-commons-configuration/xwiki-commons-configuration-api/src/main/java/org/xwiki/configuration/ConfigurationSource.java

```diff
@@ -84,6 +84,16 @@ default <T> T getProperty(String key, Class<T> valueClass, T defaultValue)
      */
     List<String> getKeys();
 
+    /**
+     * @param prefix the prefix to filter the keys
+     * @return the list of available keys in the configuration source that start with the passed prefix
+     * @since 17.4.0RC1
+     */
+    default List<String> getKeys(String prefix)
```

Comment:
Missing Unstable I guess, and same below

Reply:
Yes.